### PR TITLE
Make Constants final and static

### PR DESCRIPTION
**Pull request by sarahabimay**
@maikon
@jsuchy

### CODE REVIEW

File: test/AIPlayerTest.java [hunk 1]
```diff
@@ -1,21 +1,49 @@
...
     }
 
     private boolean isValidThrowType(Throw aThrow) {
         return aThrow.equals(Throw.PAPER) || aThrow.equals(Throw.ROCK) || aThrow.equals(Throw.SCISSORS);
     }
+
+    private class RandomFake extends Random {
+        private final double randomValue;
+
+        public RandomFake(double randomFake) {
```
Comment (jsuchy) [inline test/AIPlayerTest.java]: I'd prefer renaming randomFake to randomValue

File: test/AIPlayerTest.java [hunk 2]
```diff
@@ -1,21 +1,49 @@
...
+            this.randomValue = randomFake;
+        }
+
+        public double nextDouble() {
```
Comment (jsuchy) [inline test/AIPlayerTest.java:46]: Probably want an @Override here

**Comment by jsuchy** [comment]
Description for this PR is incomplete (describes one of the commits but not the other).

File: src/AIPlayer.java [hunk 3]
```diff
@@ -4,8 +4,14 @@
 import static java.util.Optional.of;
 
 public class AIPlayer implements Player {
+    private final int START_POSITION = 1;
+    private Random random;
     private Throw aThrow;
```
Comment (jsuchy) [inline src/AIPlayer.java]: We talked about this, but now that I'm looking at it I would probably change the name of the Throw class. Having to name variables like aThrow is annoying enough for me to find a different name.

File: src/AIPlayer.java [hunk 4]
```diff
@@ -21,6 +27,10 @@ private Throw getRandomSymbolInRange(int range) {
     }
 
     private Optional<Throw> convertToThrow(int throwIdentifier) {
+        if (throwIdentifier <= Throw.values().length){
+            Throw aThrow= Throw.values()[throwIdentifier-1];
```
Comment (jsuchy) [inline src/AIPlayer.java]: missing whitespace, also dislike the local variable as it's not providing any explanatory value.

File: src/AIPlayer.java [hunk 5]
```diff
@@ -21,6 +27,10 @@ private Throw getRandomSymbolInRange(int range) {
...
+            return of(aThrow);
+        }
         for (Throw aThrow : Throw.values()) {
             if (aThrow.equalsChoice(throwIdentifier)) {
                 return of(aThrow);
```
Comment (jsuchy) [inline src/AIPlayer.java]: There's duplication in this method, how could you remove it?

Comment (sarahabimay) [inline src/AIPlayer.java]: This was mistakenly left in.  I added another commit after this one, which I thought removed this.  Pretty sure I checked too.

**Pull request merged**